website/docs/integrations/github-actions/component-updater.md

```diff
@@ -27,34 +27,99 @@ Discover more details and a comprehensive list of `inputs` and `outputs` in the
 ### Workflow example
 
 ```yaml
-  name: "atmos-components"
-
-  on:
-    workflow_dispatch: {}
-
-    schedule:
-      - cron:  '0 8 * * 1'         # Execute every week on Monday at 08:00
-
-  permissions:
-    contents: write
-    pull-requests: write
-
-  jobs:
-    update:
-      runs-on: ubuntu-latest
-      steps:
-        - name: Update Atmos Components
-          uses: cloudposse/github-action-atmos-component-updater@v1
-          with:
-            github-access-token: ${{ secrets.GITHUB_TOKEN }}
-            max-number-of-prs: 5
-            include: |
-              aws-*
-              eks/*
-              bastion
-            exclude: aws-sso,aws-saml
+name: "Atmos Component Updater"
+
+on:
+  workflow_dispatch: {}
+
+  schedule:
+    - cron:  '0 8 * * *'  # Every day at 8am UTC
+
+jobs:
+  update:
+    environment: atmos
+    runs-on:
+      - "self-hosted"
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 1
+
+      # Install the Atmos Component Updater GitHub App:
+      # https://github.com/apps/atmos-component-updater
+      - name: Generate a token
+        id: github-app
+        uses: actions/create-github-app-token@v1
+        with:
+          app-id: ${{ secrets.ATMOS_APP_ID }}
+          private-key: ${{ secrets.ATMOS_PRIVATE_KEY }}
+
+      - name: Update Atmos Components
+        uses: cloudposse/github-action-atmos-component-updater@v2
+        env:
+          # https://atmos.tools/cli/configuration/#environment-variables
+          ATMOS_CLI_CONFIG_PATH: ${{ github.workspace }}/rootfs/usr/local/etc/atmos/
+        with:
+          github-access-token: ${{ steps.github-app.outputs.token }}
+          log-level: INFO
+          vendoring-enabled: true
+          max-number-of-prs: 10
+
+      - name: Delete abandoned update branches
+        uses: phpdocker-io/github-actions-delete-abandoned-branches@v2
+        with:
+          github_token: ${{ steps.github-app.outputs.token }}
+          last_commit_age_days: 0
+          allowed_prefixes: "component-update/"
+          dry_run: no
 ```
 
+### Requirements
+
+This action will automatically suggest pull requests in your new repository.
+To do so, we need to install a GitHub App and allow GitHub Actions to create and approve pull requests within your GitHub Organization.
+For more on the Atmos Component Updater, see [atmos.tools](https://atmos.tools/integrations/github-actions/component-updater).
```

Comment:
I don't understand, isn't that this page?

Reply:
oops. Should not have been copied over. Updating